addons/s3/requirements.txt

```diff
@@ -1,2 +1 @@
-boto==2.49.0
 boto3==1.34.60
```

Comment:
For @felliott : this is something we were concerned about but everything seems fine with Oleh's fix, at least on the OSF side; we still need to locally or staging2 test this with WB S3 (which still uses `boto`).